src/include/storage/store/table_statistics.h

```diff
@@ -34,8 +49,20 @@ class TableStatistics {
         numTuples = numTuples_;
     }
 
+    inline PropertyStatistics& getPropertyStatistics(common::property_id_t propertyID) {
+        return *(propertyStatistics.at(propertyID));
+    }
+
+    inline const TablePropertyStats& getPropertyStatistics() { return propertyStatistics; }
+
+    inline void setPropertyStatistics(
+        common::property_id_t propertyID, PropertyStatistics newStats) {
+        propertyStatistics[propertyID] = std::make_unique<PropertyStatistics>(newStats);
+    }
+
 private:
     uint64_t numTuples;
+    TablePropertyStats propertyStatistics;
```

Comment:
Though `TablePropertyStats` simply contains a primitive boolean field, it is still good to wrap it into a `unique_ptr` as I can see this data structure will become more complicated soon, and with the `unique_ptr`, we can easily avoid copying it around without notice.

Reply:
This is the TablePropertyStats alias, not the PropertyStatistics struct.